.pre-commit-config.yaml

```diff
@@ -3,5 +3,5 @@ repos:
     rev: stable
     hooks:
     - id: black
-      language_version: python3.6
+      language_version: python3.7
```

Comment:
As mentioned earlier, if you could keep it to 3.6 that would be great lol.

Reply:
Fixed.